tools/engine_tool/lib/src/logger.dart

```diff
@@ -281,11 +302,11 @@ class FlutterSpinner extends Spinner {
   });
 
   @visibleForTesting
+
```

Comment:
I think the `visibleForTesting` on 304 can be safely removed.

Reply:
Done.